pkg/resources/deployments/log-search-api.go

```diff
@@ -97,6 +100,19 @@ func logSearchAPIMeta(t *miniov1.Tenant) metav1.ObjectMeta {
 	for k, v := range t.LogSearchAPIPodLabels() {
 		meta.Labels[k] = v
 	}
+
+	// attach any labels
+	for k, v := range t.Spec.Log.Db.Labels {
```

Comment:
Here we are applying a common set of labels and annotations to both postgres server pods and logsearchapi server pods. Is this intentional? logsearchapi server is stateless and can be scheduled more freely than the postgres server container (due to pvc constraints). I am fine either ways, I am trying to clarify the intent.

Reply:
fixed